variables.tf

```
@@ -214,8 +214,20 @@ variable "enable_local_preview" {
   type        = bool
   description = "Indicates whether local preview runs can be triggered on this Stack."
   default     = false
+}
+
+variable "enable_well_known_secret_masking" {
+  type        = bool
+  description = "Indicates whether well-known secret masking is enabled."
+  default     = false
```

This feels like a case where we should build a best practice into the module and default this to `true`. What do you think?

```suggestion
  default     = true
```

---

Yes, makes sense 👍